### DISCUSSION TITLE
君子谈钱 | /cn/2017/05/money/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#930] [Comments]
https://yihui.org/cn/2017/05/money/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
美帝人力和场地太贵了，不过我还是非常想挑战一下这个系统，看看能不能把中国 R 会议的免费模式移植到美帝来，前两个月简单想了想，到头来能否实施，要看一帮同学们是否扛得住了。

> ——原帖发布于 2017-05-27 05:26:37

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我这篇文章有点狡猾，我只谈了事情的一方面，不收费是为了让想参加的都来参加、不必考虑财务问题，其实另一方面也是责任问题，收了钱再办事，责任就会更重，毕竟吃人嘴短、拿人手软。我不想让组织者背负太多的承诺。不收钱对我们来说更轻松，因为参会者也不能因为有什么服务不周的地方横加指责。

> ——原帖发布于 2017-05-28 06:34:21

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
不需要这样。下次可以早点抢着报名，明年的北京会场在北大，当然你也可以选择参加别的城市的会场。

> ——原帖发布于 2017-06-13 02:20:12

[a comment by an automated account is not shown]

**Comment by wong2** [comment]
最后的结果是怎样呢？有多少人事后退款？

**Reply by yihui** [reply]
据我所知，那次会议事后并没有人要求退款。如果需要，我可以找当年的会务组同学们确认一下，但我估计就算有，也是极个别现象。我这个首席退款官没有派上用场。
